tools/topology/topology2/include/pipelines/cavs/mixout-gain-fir-dai-copier-playback.conf

```diff
@@ -0,0 +1,144 @@
+#
+# BE playback pipeline: mixout-gain-fir-dai-copier.
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.mixout-gain-fir-dai-copier-playback.N.attribute_name"
+#
+# Usage: mixout-gain-fir-dai-copier-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.mixout-gain-fir-dai-copier-playback."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/gain.conf>
+<include/components/mixout.conf>
+<include/controls/bytes.conf>
+<include/components/eqfir.conf>
+<include/components/pipeline.conf>
+<platform/intel/efx-default.conf>
+
+Class.Pipeline."mixout-gain-fir-dai-copier-playback" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# mixout-gain-dai-copier-playback objects instantiated within the same alsaconf node must have
+		# unique pipeline_id attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+		mixout."1" {}
+		copier."1" {
+			type dai_in
+			node_type $HDA_LINK_OUTPUT_CLASS
+			period_sink_count 0
+			period_source_count 2
+			num_audio_formats 2
+
+			# copier only supports one format based on mixin/mixout requirements: 32-bit 48KHz 2ch
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	24
+				out_bit_depth		32
+				out_valid_bit_depth	24
+				dma_buffer_size "$[$obs * 2]"
+			}
+
+			Object.Base.audio_format.2 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$obs * 2]"
+			}
+		}
+		gain."1" {
+			num_audio_formats 2
+
+			# 32-bit 48KHz 2ch
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	24
+				out_bit_depth		32
+				out_valid_bit_depth	24
+			}
+
+			Object.Base.audio_format.2 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+			}
+		}
+		eqfir."1" {
+			num_audio_formats 2
+
+			# 32-bit 48KHz 2ch
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	24
+				out_bit_depth		32
+				out_valid_bit_depth	24
+			}
+
+			Object.Base.audio_format.2 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+			}
+
+			Object.Control.bytes."1" {
+				IncludeByKey.EFX_FIR_PARAMS {
```

Comment:
@juimonen @aiChaoSONG has a patch coming up for including small snippets of conf nodes without including a new file. We should wait until that is merged

Reply:
so we actually want to have the eq blobs in separate files as they can be produced with octave tools.